samples/can/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _can-sample:
+
+Controller Area Network
+###########
+
+Overview
+********
+
+This sample demonstrates how to use the CAN API.
```

Comment:
On first use, would be better to spell out the acronym:

    This sample demonstrates how to use the Controller Area Network (CAN) API.